common/src/main/kotlin/com/gitlab/kordlib/common/Color.kt

```diff
@@ -39,4 +39,4 @@ private fun rgb(red: Int, green: Int, blue: Int): Int {
             (blue and 0xFF) shl 0
 }
 
-val java.awt.Color.kColor get() = Color(rgb)
\ No newline at end of file
+val java.awt.Color.kColor get() = Color(rgb and 0x00FFFFFF)
```

Comment:
This should be a part of the `rgb` function as we expect the returned color to have no alpha for both the conversion method and the constructor.